.github/workflows/build.yml

```diff
@@ -100,3 +109,60 @@ jobs:
       - name: Build Dasharo
         run: |
           ./build.sh ${{ matrix.model }}
+  test_msi:
+    runs-on: self-hosted
+    strategy:
+      matrix:
+        vendor: [ msi ]
+        model: [ ms7d25_ddr5 ]
```

Comment:
We have two more variants to test

Reply:
Does it mean it has to be all in a single MR? Why not merge a single board first to see how it works in practice, and add more later on? We will be able to gather problems/failures and could discuss how to resolve these already.

We've asked to move from github actions anyway, so maybe we will do this as well then.

Reply:
Then I'd ask you to improve it, please, to meet your requirements.

Reply:
Added all models we have in lab: https://github.com/Dasharo/coreboot/pull/432/commits/bf4b3d93192b5778c9a0d6ca75b7a5edf4cb8228#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R112